ROADMAP.md

```diff
@@ -1,62 +1,191 @@
+- We are alerted when an interop test starts failing on one of our client repositories, and we can dispatch the alert to repo maintainers.
+
+#### 2. Design process for adding new tests
+<!-- TODO: Assign a quarter -->
+<!-- TODO: Create issue -->
+We have an explicit, working, Design Process for adding new tests
+- The design is documented in `./DESIGN.md`.
+- The design is followed by the team when we add new features.
+- There is a clear path when it comes to testing new features. This might mean testing multiple `masters` against each other.
+
+#### 3. Be the home for all interop tests
+<!-- TODO: Assign a quarter -->
+<!-- TODO: Create issue -->
+We have ported the tests from `libp2p/interop`
+- This repository implement tests `connect`, `dht`, `pubsub`, `steam` ([ref](https://github.com/libp2p/interop/blob/ce0aa3749c9c53cf5ad53009b273847b94106d40/src/index.ts#L32-L35))
+- At of writing (2022-09-27), it is disabled in `go-libp2p` ([ref](https://github.com/libp2p/go-libp2p/actions/workflows/interop.yml)), and it is used in `js-libp2p` ([ref](https://github.com/libp2p/js-libp2p/actions/runs/3111413168/jobs/5050929689)).
+
+#### 4. Stable build process
+<!-- TODO: Assign a quarter -->
+<!-- TODO: Create issue -->
+We have a more stable build process that doesn't risk breaking
+- We generate artifacts for old versions during merges to the libp2p repositories https://github.com/libp2p/test-plans/issues/35#issuecomment-1254991985
+
+### [C. Future-proof Benchmarking](https://github.com/libp2p/go-libp2p/issues/1810)
+**Why**: For libp2p to be competitive, it needs to delivers comparable performance to widely used protocols on the internet, namely HTTP/2 and HTTP/3.
+
+**Goal**: We have a test suite that runs libp2p transfers between nodes located at different locations all over the world, proving that libp2p is able to achieve performance on par with HTTP. The test suite is run on a continuous basis and results are published to a public performance dashboard.
+
+#### 1. [Benchmarking using nix-builders](https://github.com/testground/testground/pull/1425)
+- (Dependency: remote machines need Nix installed)
+#### 2. Benchmarking using remote runners
+Benchmarking using first class support for remote runners (using `remote:exec`) in Testground
+
+### D. Expansive protocol test coverage
+<!-- TODO: Assign a quarter -->
+<!-- TODO: Create issue -->
+**Why:** Having interoperability tests with lots of transports, encryption mechanisms, and stream muxers is great. However, we need to stay backwards-compatible with legacy libp2p releases, with other libp2p implementations, and less advanced libp2p stacks.
+
+**Goal:** We have expansive test coverage that covers all protocols.
+<!-- TODO: List all major protocol test backlog items here.
+Decide as a team which ones to prioritize and then assign to quarters.-->
+#### 1. AutoNat
+#### 2. Hole Punching
+#### 3. Custom topologies
+#### 4. MTU Fixes
```

Comment:
That's an issue with Testground's network simulation. It needs to be fixed in Testground, we can't fix it in a libp2p test plan.

Reply:
Agreed.  We should make clear that it has a dependency on Testground.